src/Microsoft.Azure.SignalR/ClientInvocation/CallerClientResultsManager.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using Microsoft.Azure.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal sealed class CallerClientResultsManager : ICallerClientResultsManager, IInvocationBinder
+    {
+        private readonly ConcurrentDictionary<string, PendingInvocation> _pendingInvocations = new();
+        private readonly ConcurrentDictionary<string, List<string>> _serviceMappingMessages = new();
+        private readonly string _serverGUID = Guid.NewGuid().ToString();
```

Comment:
When it is an instance property, calling it `_serverGuid` is not accurate. Maybe clientResultManagerId? It just happens to be the same for the server if you are using `CallerClientResultsManager` as a singleton. Or make it a static property.

Reply:
renamed

Reply:
done